Example/PaymentSheet Example/PaymentSheetUITest/PaymentSheetUITest.swift

```diff
@@ -799,6 +798,38 @@ extension PaymentSheetUITest {
         XCTAssertTrue(cardButton.waitForExistence(timeout: 10.0))
         cardButton.forceTapElement()
 
+        // Fill out billing details if required
+        let addBillingDetailsButton = applePay.buttons["Add Billing Address"]
```

Comment:
do we do this elsewhere?  I'm not finding it.

Reply:
I can't find it anywhere, but that was my best guess as to how it passed in the first place...

Reply:
Somehow the simulator had a billing address attached to Apple Pay when CI passed